src/client/components/Pagination/Pagination.component.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useRef, useEffect } from 'react';
+import './Pagination.styles.css';
+
+function Pagination({ num }) {
+  const [floor, setFloor] = useState(num);
+  const level0 = useRef();
+  const level1 = useRef();
+  const level2 = useRef();
+  const level3 = useRef();
+  const level4 = useRef();
+  const level5 = useRef();
```

Comment:
Why are you using refs here?

Reply:
nice use of useRef Dlnya, good that you have learned it :)

Reply:
Thank you @Enqira  , I find it very useful :)